src/general/MainMenu.cs

```diff
@@ -622,7 +622,8 @@ private void OnReturnFromArtGallery()
 
     private void OnWebsitesButtonPressed()
     {
-        websiteButtonsContainer.Visible = !websiteButtonsContainer.Visible;
+        websiteButtonsContainer.ShowModal();
+        websiteButtonsContainer.RectSize = Vector2.Zero;
```

Comment:
Presumably this hack is needed because this doesn't use our custom dialog? I think a comment should be added here as to why this is needed.